README.md

```diff
@@ -175,19 +175,17 @@ The output will be in `target/html/master.html`.
 
 See [benchmarking.md](benchmarking.md) for information on running basic performance tests for this proxy.
 
-### GitHub action for performance
+### Jenkins pipeline for performance
 When a PR is created and the performance tests are needed, the following comment shall be added into the PR:
 
 ```
-/perf
+@strimzi-ci run perf
 ```
 
-It will launch the `Performance` build, compare the results with the previous execution and store the results in the gh-pages repo under the `.benchmark` directory.
+It will launch the [kroxylicious-performance-tests-pr](https://master-jenkins-csb-amq-streams.apps.ocp-c1.prod.psi.redhat.com/job/kroxylicious/job/kroxylicious-performance-tests-pr/) build, that will insert a comment with a summary into the PR comparing the results with the previous execution.
```

Comment:
Is that Jenkins instance Red Hat internal? If so we supposed to link to externally?

Reply:
I believe this link can be seen for anybody internal or external, but only manipulated by authorized ones.

Reply:
That name does not resolve in public DNS.